manim/mobject/types/vectorized_mobject.py

```diff
@@ -276,7 +298,7 @@ def get_style(self, simple=False):
 
         return ret
 
-    def match_style(self, vmobject, family=True):
+    def match_style(self, vmobject, family=True, recurse=True):
```

Comment:
Why is `recurse` added if it's seemingly not used?

Reply:
b80e6b35a259e58f8b122e23a8cf380d3a2c4044